### PR TITLE
:memo: add FAQ content

### PR DESCRIPTION
**Pull request by dustingooding**
Problem:
- There is no FAQ content.

Solution:
- Add some.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
-# FAQ
+# Frequently Asked Questions
 
-### What is Project Beman?
```
Comment (bretbrownjr) [inline docs/FAQ.md:3]: I think "What is Project Beman?" is maybe the most frequently asked question, but maybe that's another PR.

File: docs/FAQ.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+No! The Beman Project wants to work with you, with experience, feedback, and expertise. Obviously, a paper must be written before becoming accepted as part of the standard, but that paper can be written in tandem with your library development.
 
-### What is the relationship between Project Beman and Boost Foundation?
+In fact, there are plenty of papers already written that need an implementation before they can progress in through the standards track. You can contribute to the Beman project by finding a paper that interests you and working on its library.
 
+Please reach out to the Beman Project Leads to help get your specific project started.
 
+## I have a crazy library that’s probably never going to be a standard library feature, but I use it and think it’s neat. Can I contribute it to the Beman Project?
+
+We’re glad you’ve developed a neat and useful library, but the Beman Project is dedicated to getting the highest quality libraries through the rigorous effort of standardization.
```
Comment (bretbrownjr) [inline docs/FAQ.md:17]: I think rewording this Q&A to remove words like "neat" and "crazy" would be appropriate. I could see a reader taking this as not being worded in good faith.

It might also be worth pointing the reader at the Discourse. Hashing out library viability in an interactive context where people can get to know one another seems helpful.

**Review by bretbrownjr — CHANGES_REQUESTED** [review]
I like having an FAQ.

I'd reword the "crazy library" question before landing.

**Review by rmpowell77 — COMMENTED** [review]
I think this is a great improvement.  I see this as a work in progress, so let's merge this in.

File: docs/FAQ.md (hunk 2 above)
Comment (camio) [reply docs/FAQ.md:17]: @bretbrownjr I attempted to address this concern in https://github.com/beman-project/beman/pull/20. LMK what you think there.
